docs/app/Examples/modules/Popup/Variations/PopupOffsetExample.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Button, Icon, Popup } from 'semantic-ui-react'
+
+export default function PopupOffsetExample() {
+  return (
+    <div>
+      <Popup
```

Comment:
We can remove the extra `div` wrapper here and just return the `<Popup />`.  As long as a single element is returned, React is happy.  React only requires an extra containing element (such as a `div`) if multiple siblings need to be returned.